package.json

```diff
@@ -17,7 +18,8 @@
     "format": "prettier --loglevel warn --write '**/*.ts' && tslint --fix --format stylish '**/*.ts' --exclude 'node_modules/**'",
     "test": "npm run lint && npm run unit-test",
     "unit-test": "tap test/system test/lib -R=spec --timeout=300",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "prepublish": "tsc"
```

Comment:
I think it runs `prepare` during `prepublish` anyway?

Reply:
Correct, removed it!